omim2obo/parsers/omim_txt_parser.py

```diff
@@ -18,9 +19,19 @@
 
 
 LOG = logging.getLogger('omim2obo.parser.omim_titles_parser')
-
-
-def retrieve_mim_file(file_name: str, download=False, return_df=False) -> Union[List[str], pd.DataFrame]:
+# MIM_NUMBER_DIGITS = 6  # if you see '6' in a regexp, this is what it refers to
+MORBIDMAP_PHENOTYPE_MAPPING_KEY_MEANINGS = {
```

Comment:
@cmungall @matentzn If there is one of these mapping keys in a `morbidmap.txt` Phenotype label, we are now adding tripe (`MIM#` `biolink:GeneToDiseaseAssociation` `biolink:has_evidence`). However, I imagine we also want to add the kind of evidence itself? Not sure if we want to keep the "mapping key" (1 through 4), but the mapping key description, we should probably add some how. I imagine this involves me adding some kind of axiom with annotation properties? This a more advanced thing than I've typically done  in RDF. Wondering what you guys suggest?

Reply:
Discussed this with Nico, Kevin Schaper, and Sabrina today. Looks like koza is fetching mostly the same type of info here, except that there has been some configuration to load up some manually curated ECO mappings on each of the 4 OMIM phenotype mappings. Nico came to determination that I can add the literal text of the phenotype mapping descriptions, and that will be fine for now.